addons/editor-square-inputs/addon.json

```diff
@@ -21,6 +21,12 @@
       "type": "boolean",
       "default": true
     },
+    {
+      "name": "Number inputs",
+      "id": "number",
+      "type": "boolean",
+      "default": false
```

Comment:
Tiny nitpick but the default is a disabled setting in between 2 enabled ones. Could it be moved down so the enabled ones go first?